src/cmd/create.go

```diff
@@ -265,8 +265,19 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 	logrus.Debugf("%s canonicalized to %s", currentUser.HomeDir, homeDirEvaled)
 	homeDirMountArg := homeDirEvaled + ":" + homeDirEvaled + ":rslave"
 
+	bootMountFlags := "rw"
```

Comment:
I am curious why the logic here for `/boot` is subtly different from that of `/usr` below. Is there something inherently different in the two cases, or is it just a cosmetic difference?

If it's just a cosmetic issue, then it will be good to keep them consistent. The code around mount points and options sometimes requires careful attention while reading, so there's some value in avoiding superficial differences. :)

(For what it's worth, in the absence of a more profound argument, starting from a more conservative "default" value of `ro` and then conditionally switching to `rw` feels safer.)

Reply:
See https://github.com/containers/toolbox/pull/804

Reply:
You've got a point. Good catch! I'm not sure after such a long time what made me write it like so. I'll keep it in mind in the future.